numpydoc/docscrape_sphinx.py

```diff
@@ -219,6 +219,24 @@ def _obj(self):
             return self._f
         return None
 
+    def _get_class_that_defined_method(self, method):
```

Comment:
Perhaps some reasoning here, why is this function needed?

Reply:
Okay, I'll guess my way forward :-)